luaui/Widgets/gui_gridmenu.lua

```diff
@@ -788,6 +789,9 @@ function widget:ViewResize()
 end
 
 function reloadBindings()
+	-- initialise keySymCharsReverse from keySymChars
+	Cfgs.keySymCharsReverse = table.invert(Cfgs.keySymChars)
```

Comment:
You can make this a local and remove another assignment from Cfgs since this is not used anywhere else, it's a cheap operation for a seldom invoked function like this